spec/e2e/default_install_spec.rb

```diff
@@ -19,10 +19,10 @@ def browser
 
   describe 'default webpage' do
     it 'redirects from root' do
-      browser.goto ctr_base_url
+      browser.goto 'http://localhost:80/'
```

Comment:
I think it will need to be 8081 based on the external port you assigned the container.

Reply:
Actually nevermind, I read the container port numbers backwards